////////////////////////////////////////////////////////////////////////////////
/// @HdmiEdid.proto
///
/// Copyright 2018 Bose Corporation
////////////////////////////////////////////////////////////////////////////////

package DataCollection;

message HdmiEdid
{
  required string eedid = 1;
}